cmd/goci/main.go

```diff
@@ -32,10 +38,6 @@ func main() {
 }
 
 func run(mode string) error {
-	if strings.Contains(environment.Branch, "/") {
```

Comment:
We will still want validate to run in all modes, but after the case statement below.